### DISCUSSION TITLE
Iintegration with maven spring boot project.

### DISCUSSION THREAD

**Discussion by f3rchoc** [Xray-App/xray-junit-extensions#37] [Q&A]
Hi there. Are there considerations for using xray-junit-extensions in a spring boot project? I think so exits conflict with its spring-boot-starter-test dependency.
Thanks for your help.

**Comment by bitcoder** [comment]
Please provide an example and a concrete error you're getting to be able to help you out.

**Reply by bitcoder** [reply]
Additionally, I've never tried this project together with sprint boot. But it would help if you can provide an example of a small project, showcasing the error you're getting

**Comment by bitcoder** [comment]
This should be fixed on the latest 0.7.1 just released: https://github.com/Xray-App/xray-junit-extensions/releases/tag/0.7.1 
Thanks for reporting
